### ISSUE TITLE
Linux compressonatorcli (4.2.5185) fails to load JPG files

### ISSUE THREAD

**Issue by Iyeyasu** [GPUOpen-Tools/compressonator#205]
The `compressantorcli` tool for Linux from [releases ](https://github.com/GPUOpen-Tools/compressonator/releases/tag/V4.2.5185) fails to properly load JPG files. This is observed on the following platforms:

- Ubuntu 22.04 LTS running on Windows WSL
- Ubuntu 20.04 LTS running on Azure DevOps pipeline

The following commands both seem to fail during call to `AMDLoadMIPSTextureImage()`:

```
> compressonatorcli -fd ETC2_RGB EV_BODY_MR.jpg EV_BODY_MR.dds
Error: loading source image
```

```
> compressonatorcli -imageprops EV_BODY_MR.jpg
File Name: EV_BODY_MR.jpg
File Full Path: <redacted>
File Size: 1 MB
Error: reading image, data type not supported.
Error: Failed to retrieve image properties
```

**Comment by YGaurora** [comment]
They have forgeted put qt.conf in this package.You can write a config and make a plugin directory.just like this. running on Windows WSL too.
![NgVk5ij8np](https://user-images.githubusercontent.com/31273266/183397634-75843738-49ee-4b87-919a-6d8ac0a67069.jpg)
![image_](https://user-images.githubusercontent.com/31273266/183397718-72b4ee60-6baf-4739-bced-b2783c862e74.jpg)
![NgVk5ij8np](https://user-images.githubusercontent.com/31273266/183397727-0024578d-8e81-4118-9e39-da1c7f930f92.jpg)

**Comment by dsleep** [comment]
I have a push request related to this, basically just an oversight to require QT.

**Comment by NPCompress** [comment]
@dsleep  Thanks for the qt.conf issue fix.

**Comment by NPCompress** [comment]
@dsleep Issue fixed in next release v4.3
